features/project_test.feature

```diff
@@ -0,0 +1,286 @@
+@api @test
+Feature: Test
+
+  Users should be able to retrieve the test details by its id.
+
+  The details of a test contains:
+  - name
+  - category
+  - tags
+  - key
+  - project (object)
+  - author/contributors
+  - number of results
+  - first run date
+  - last run date
+  - latest project version
+  - passing (last result)
+  - active/inactive
+
+
+
+  Scenario: An organization member should be able to get the test details without project data in his organization
```

Comment:
In the description of the default case, I would not mention was is not there (i.e. "without project data"). Otherwise you would also need to add "without contributions" and all other absent filters that we may add.